### PR TITLE
Added non-data model hyperparameters for H2O AutoML

### PR DESCRIPTION
**Pull request by vopani**
For #27 

Note: This doesn't include data parameters

### CODE REVIEW

File: h2o_wave_ml/ml.py [hunk 1]
```diff
@@ -133,17 +133,39 @@ def predict(self, data: Optional[List[List]] = None, file_path: str = '', **kwar
 
         """
 
-    @abc.abstractproperty
+    @property
```
Comment (geomodular) [inline h2o_wave_ml/ml.py:136]: Ahh, yes thanks. I forgot this is the proper way now.

**Review by geomodular — APPROVED** [review]
I set `MAX_RUNTIME_SECS` and `MAX_MODELS` to some; I thought, reasonable values. Do you think it's ok to omit them?

**Comment by vopani** [comment]
The runtime default is 1 hour anyway. It really doesn't matter much but I think it's safer to use a library's default (since a user would expect that if not explicitly set), rather than some additional intelligence in WaveML.

**Review by geomodular — APPROVED** [review]
Looks great, thanks @vopani!

**Pull request merged**